docs/raspberry-pi-setup.md

```diff
@@ -0,0 +1,457 @@
+| **Heat Sink** | Aluminum with thermal pad | Temperature management |
+
+### MVP Prototype Hardware (4 Plant Modules)
+
+| Hardware Module | Component | GPIO Connection |
+|-----------------|-----------|-----------------|
+| **Sensors** | 4x Capacitive Soil Moisture | MCP3008 ADC (SPI) |
+| | TSL2591 Light Sensor | I2C (GPIO 2/3) |
+| | DHT22 Temp/Humidity | GPIO 4 |
+| | pH/EC Sensor | MCP3008 ADC (SPI) |
+| **Actuators** | 12V Water Pump | GPIO 20 (Relay) |
+| | 4x 5V Water Valves | GPIO 21-24 (Transistor) |
+| | 24V LED Strip (PWM) | GPIO 18 (Hardware PWM) |
+| **Display** | Status RGB LEDs | GPIO 5, 6, 13 |
+| | Optional OLED Display | I2C (GPIO 2/3) |
+
+## 🚀 Quick Start Installation
+
+### Option 1: Automated Setup (Recommended)
+
+```bash
+# 1. Flash Raspbian OS Lite to SD card
+# 2. Boot Pi and enable SSH
+# 3. Copy repository to Pi
+git clone https://github.com/andrekirst/plantwall.git
+cd plantwall/src/plant-wall-control
+
+# 4. Run master setup script
+sudo ./scripts/setup-raspberry-pi.sh
+
+# 5. Deploy application
+sudo ./systemd/install-service.sh
+
+# 6. Validate hardware
+sudo ./scripts/hardware-test.sh
+```
+
+### Option 2: Manual Setup
+
+Follow the detailed steps in the sections below.
+
+## 🔧 Detailed Setup Process
+
+### Step 1: OS Installation and Initial Setup
+
+#### 1.1 Flash Raspbian OS Lite
+
+```bash
+# Download Raspbian OS Lite (latest)
+# Flash to 32GB Industrial MicroSD card using Raspberry Pi Imager
+
+# Enable SSH and WiFi during imaging:
+# - Enable SSH with public key authentication
+# - Configure WiFi credentials
+# - Set hostname: plantwall-001
+```
+
+#### 1.2 First Boot Configuration
+
+```bash
+# SSH into the Pi
+ssh pi@plantwall-001.local
+
+# Update system
+sudo apt update && sudo apt full-upgrade -y
+
+# Configure timezone and locale
+sudo raspi-config nonint do_change_timezone Europe/Berlin
+sudo raspi-config nonint do_change_locale de_DE.UTF-8
+
+# Expand filesystem
+sudo raspi-config nonint do_expand_rootfs
+```
+
+### Step 2: Hardware Interface Setup
+
+#### 2.1 Enable Hardware Interfaces
+
+```bash
+# Enable GPIO, I2C, SPI interfaces
+sudo raspi-config nonint do_i2c 0        # Enable I2C
+sudo raspi-config nonint do_spi 0        # Enable SPI
+sudo raspi-config nonint do_ssh 0        # Enable SSH
+sudo raspi-config nonint do_camera 1     # Disable camera (save resources)
+
+# Configure GPU memory split for headless operation
+echo "gpu_mem=16" | sudo tee -a /boot/config.txt
+
+# Enable hardware PWM on GPIO 18
+echo "dtoverlay=pwm,pin=18,func=2" | sudo tee -a /boot/config.txt
+```
+
+#### 2.2 Install Required Packages
+
+```bash
+# System packages
+sudo apt install -y \
+    git curl wget \
+    i2c-tools spi-tools \
+    build-essential \
+    htop iotop \
+    fail2ban ufw \
+    logrotate \
+    python3-pip
+
+# Install log2ram for SD card longevity
+wget -O log2ram.deb https://github.com/azlux/log2ram/releases/latest/download/log2ram_1.6.1_all.deb
```

Comment:
Da kommt bei mir ein 404-Fehler:
-2025-08-20 22:52:24--  https://github.com/azlux/log2ram/releases/latest/download/log2ram_1.6.1_all.deb
Resolving github.com (github.com)... 140.82.121.3
Connecting to github.com (github.com)|140.82.121.3|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://github.com/azlux/log2ram/releases/download/1.7.2/log2ram_1.6.1_all.deb [following]
--2025-08-20 22:52:24--  https://github.com/azlux/log2ram/releases/download/1.7.2/log2ram_1.6.1_all.deb
Reusing existing connection to github.com:443.
HTTP request sent, awaiting response... 404 Not Found
2025-08-20 22:52:24 ERROR 404: Not Found.